src/network/networkd-json.c

```diff
@@ -20,6 +20,7 @@
 #include "networkd-route-util.h"
 #include "networkd-route.h"
 #include "networkd-routing-policy-rule.h"
+#include "sd-dhcp-client-id.h"
```

Comment:
Please drop this. I think it is not necessary to include the header explicitly here.